chapters/unitexpressions.tex

```diff
@@ -42,12 +42,17 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
   unit-operand [ unit-exponent ]
 
 unit-exponent :
-   [ "+" | "-" ] UNSIGNED-INTEGER
+   [ "+" | "-" ] ( UNSIGNED-INTEGER | "(" UNSIGNED-INTEGER "/" UNSIGNED-INTEGER ")" )
 \end{lstlisting}
 
 The SI standard uses super-script for the exponentation, and does thus not define any operator symbol for exponentiation.
-A \lstinline[language=grammar]!unit-factor! consists of a \lstinline[language=grammar]!unit-operand! possibly suffixed by a possibly signed integer number, which is interpreted as an exponent.
+A \lstinline[language=grammar]!unit-factor! consists of a \lstinline[language=grammar]!unit-operand! possibly suffixed by a possibly signed integer or rational number, which is interpreted as an exponent.
```

Comment:
I just noticed that there's a bit too much of _possibly_ here.  It get's easier to formulate a clear sentence if we don't talk about the signedness at all:
```suggestion
A \lstinline[language=grammar]!unit-factor! consists of a \lstinline[language=grammar]!unit-operand!, optionally suffixed by an integer or rational number which is interpreted as an exponent.
```

Reply:
I agree that it is clumsy, but it seems it is intended to by a copy of the grammar; so that each "possibly" correspond to an optional variant in the grammar. Seems best to discuss separately.